mxcube3/ui/actions/queue.js

```diff
@@ -402,7 +400,7 @@ export function addTask(sampleIDs, parameters, runNow) {
                      checked: true };
 
       if (!state.queue.queue.includes(sampleID)) {
-        const sample = state.sampleGrid.sampleList[sampleID];
+        const sample = Object.assign({}, state.sampleGrid.sampleList[sampleID]);
```

Comment:
@meguiraun: why do we need a copy here? (just a small question - if there is a copy here it may indicate our corresponding reducer is wrong, ie. not returning a real new state)